skunkworks_crow/src/main/java/org/odk/share/views/ui/about/AboutActivity.java

```diff
@@ -22,6 +25,7 @@
 
     private static final String LICENSES_HTML_PATH = "file:///android_asset/open_source_licenses.html";
     private static final String USER_GUIDE_HTML_PATH = "file:///android_asset/user_guide.html";
+    static final String url = "https://github.com/opendatakit/skunkworks-crow/blob/master/README.md";
```

Comment:
you can make this private and name the variable something like `SKUNKWORKS_CROW_README_URL` instead of `url`